spec/requests/api/collections_spec.rb

```diff
@@ -119,6 +119,11 @@ def test_collection_bulk_query(collection, collection_url, klass, id = nil)
       test_collection_query(:flavors, flavors_url, Flavor)
     end
 
+    it "query FloatingIps" do
```

Comment:
This test is duplicating some of the tests in the other spec file. If this wasn't obvious then I'd suggest keeping the other ones, but it's up to you 😄